Source/SA/DPD/DPD.cs

```diff
@@ -0,0 +1,204 @@
+﻿using NationalInstruments.RFmx.InstrMX;
+using NationalInstruments.RFmx.SpecAnMX;
+using NationalInstruments.ModularInstruments.NIRfsg;
+using NationalInstruments.ModularInstruments.NIRfsgPlayback;
+using static NationalInstruments.ReferenceDesignLibraries.SG;
+using System;
+
+namespace NationalInstruments.ReferenceDesignLibraries.SA
+{
+    public static class DPD
+    {
+        #region Type_Definitionss
+        public struct CommonDPDConfiguration
+        {
+            public double dpdMeasurementInterval;
+            public RFmxSpecAnMXDpdMeasurementSampleRateMode dpdSampleRateMode;
+            public double sampleRate;
```

Comment:
add unit

Reply:
@cindyxie thanks for adding a comment with the unit. However, Sean meant to add the unit to the variable name. In C#, the variable should have the unit in the name with an underscore. Hence "dpdMeasurementInterval_s". In LabVIEW it would be "DPD Measurement Interval (s)".